src/include/distributed/multi_physical_planner.h

```diff
@@ -247,6 +249,6 @@ extern bool TaskListMember(const List *taskList, const Task *task);
 extern List * TaskListDifference(const List *list1, const List *list2);
 extern List * TaskListUnion(const List *list1, const List *list2);
 extern List * FirstReplicaAssignTaskList(List *taskList);
-
+extern List * AssignAnchorShardTaskList(List *taskList);
```

Comment:
Minor, I think `FirstReplicaAssignTaskList` should be below `AssignAnchorShardTaskList` wrt definition orders in the source.